force/sobjects.go

```diff
@@ -15,9 +15,8 @@ type SObject interface {
 
 // Response received from force.com API after insert of an sobject.
 type SObjectResponse struct {
-	Id      string    `force:"id,omitempty"`
-	Errors  ApiErrors `force:"error,omitempty"` //TODO: Not sure if ApiErrors is the right object
-	Success bool      `force:"success,omitempty"`
+	Id     string    `force:"id,omitempty"`
+	Errors ApiErrors `force:"error,omitempty"` //TODO: Not sure if ApiErrors is the right object
```

Comment:
So the error was happening because of the `Success` key?

Reply:
success was never assigned. this is to get rid of the var that's never used and is misleading